server/src/web/express.ts

```
@@ -0,0 +1,72 @@
+
+import { Context } from '../config/context';
+import {
+  incomingRequestLogger,
+  makePerRequestErrorLogger,
+  makePerRequestRouteLogger,
+} from '../web/routes/routeLogger';
+import { buildApi, listRoutes } from './api';
+import { contextMiddleware } from './routes/router-helpers';
+
+const storeRawBodyIntoRequest = (req: any, _res: any, buf: Buffer) => {
+  if (buf && buf.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)